src/modules/solution-explorer/solution-explorer-panel/solution-explorer-panel.ts

```diff
@@ -264,7 +271,7 @@ export class SolutionExplorerPanel {
     this.removeSolutionFromSolutionHistroy(solutionUri);
   }
 
-  public selectProtocol(protocol: string): void {
+  public selectProtocol(protocol: SupportedProtocols | string): void {
```

Comment:
Würde den Typ des parameters zu beschränken, damit z.B. `selectProtocol('foobar');` zu Fehlern führt:

```suggestion
  public selectProtocol(protocol: SupportedProtocols): void {
```
 
Oder gibt es Gründe, die dagegen sprechen?

Reply:
In der selectRemoteSolution wird dort ein string übergeben, der aus einem vorhandenen string ausgelesen wird... Aber es spricht eigentlich nichts dagegen die Funktion ein wenig umzubauen 🤔